crates/bitwarden-wasm-internal/README.md

```diff
@@ -2,7 +2,13 @@
 
 **Note:** This is only for internal use. Bitwarden will not provide any support for this crate.
```

Comment:
Should we remove this line from here now that the disclaimer is linked in the crate?

Reply:
This is show on NPM so I figured the disclaimer still serves a purpose.

Reply:
Ah yeah, forgot about NPM, good point